overcloud.yml

```diff
@@ -25,7 +25,7 @@
       - name: generate os-net-config-mappings.yaml
         template:
           src: "os-net-config-mappings.yaml.j2"
-          dest: "/home/stack/os-net-config-mappings.yaml"
+          dest: "/home/stack/extra_templates/os-net-config-mappings.yaml"
```

Comment:
you have to create the `extra_templates` folder before this task if the folder does not exist.